src/ocean/util/encode/Base64.d

```diff
@@ -503,10 +508,38 @@ unittest
     // decodeTest
     {
         ubyte[1024] decoded;
-        ubyte[] result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw==", decoded);
+        Const!(ubyte)[] result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw==", decoded);
```

Comment:
because of the reassigning it here: https://github.com/sociomantic-tsunami/ocean/pull/156/files/7ba30bf6d7030f4220b2eeaeaf443ad324444622#diff-183a2b58e80d01cce23dd21bd51c86ceR521

Not terribly important, though, I can try to please you - I was just fixing the d2 failure.